### PR TITLE
change tags for better data support

### PR DESCRIPTION
**Pull request by dicko2**

### CODE REVIEW

File: src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs [hunk 1]
```diff
@@ -22,33 +22,23 @@ public static Task OnConsumeError(IMessageContext eventContextMessageContext, Ex
...
         eventContextMessageContext.Items.TryGetValue("timer", out var timer);
         var theTimer = (Stopwatch)timer;
         theTimer.Stop();
-        telemetryClient.TrackDependency("Kafka-Consumer",
-            eventContextMessageContext.ConsumerContext.Topic,
-            eventContextMessageContext.ConsumerContext.Partition.ToString(),
-            eventContextMessageContext.ConsumerContext.Offset.ToString(),
-            DateTimeOffset.UtcNow,
-            theTimer.Elapsed, "500", false);
+        telemetryClient.TrackKafkaDependency(eventContextMessageContext, "500", false, theTimer.Elapsed);
```
Comment (virasak) [inline src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs]: Should we have specific error code for this? maybe we can start by the name of Exception type

Comment (virasak) [reply src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs]: "500"

File: src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs [hunk 2]
```diff
@@ -22,33 +22,23 @@ public static Task OnConsumeError(IMessageContext eventContextMessageContext, Ex
...
         var theTimer = (Stopwatch)timer;
         theTimer.Stop();
-        telemetryClient.TrackDependency("Kafka-Consumer",
-            eventContextMessageContext.ConsumerContext.Topic,
-            eventContextMessageContext.ConsumerContext.Partition.ToString(),
-            eventContextMessageContext.ConsumerContext.Offset.ToString(),
-            DateTimeOffset.UtcNow,
-            theTimer.Elapsed, "200", true);
+        
+        telemetryClient.TrackKafkaDependency(eventContextMessageContext, "200", true, theTimer.Elapsed);
```
Comment (virasak) [inline src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs]: "200" -> "ok"?

**Review by virasak — APPROVED** [review]

File: src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs (hunk 1 above)
Comment (dicko2) [reply src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs]: fixed

File: src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs (hunk 2 above)
Comment (dicko2) [reply src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs]: fixed

**Pull request merged**